jsapp/js/account/usage/usageContainer.tsx

```diff
@@ -88,18 +88,18 @@ const UsageContainer = ({
           <data>{limitDisplay(usage)}</data>
         </li>
         {isStripeEnabled && (
-          <li>
-            <strong>{t('Balance')}</strong>
-            <strong
-              className={cx({
+          <li className={cx(styles.balanceEntry)}>
+            <label>{t('Balance')}</label>
```

Comment:
This looks good, thank you! The only thing I would ask that you change is making the 'Balance' text bold again, just to match the [Figma design](https://www.figma.com/file/NonWWcfadc3QfS3q3D8hRB/%E2%9A%99%EF%B8%8F-KPI-%2F-Account-Settings?type=design&node-id=2580-27554&mode=design&t=oVP2mXoTfyzjw3c8-0):

![image](https://github.com/kobotoolbox/kpi/assets/7819986/5ab9586d-75e2-481b-a004-44fb047135f5)

But keep the `<label/>` tag, since I think it's better semantically 👍